Stack/Opc.Ua.Core/Stack/Transport/TransportListenerSettings.cs

```diff
@@ -117,6 +117,17 @@ public bool ReverseConnectListener
             get { return m_reverseConnectListener; }
             set { m_reverseConnectListener = value; }
         }
+
+        /// <summary>
```

Comment:
The only case I think of is if the user got used to binding all the NIC's when an IP has been specified instead of a hostname, but this should not be encountered so often, therefore I agree we can use the assumption mentioned above.